include/stddef.h

```diff
@@ -32,7 +32,7 @@
 typedef long ptrdiff_t;
 #ifndef _SIZE_T
 #define _SIZE_T
-typedef long long size_t;
+typedef unsigned long size_t;
```

Comment:
No, those both means only one thing: size_t was wrong. It must be unsigned long (64) and ssize_t signed long. stddef.h is temporary, in fact it's generated by compiler... when we have native one.